src/govuk/components/notification-banner/notification-banner.yaml

```diff
@@ -0,0 +1,177 @@
+params:
+- name: text
+  type: string
+  required: true
+  description: If `html` is set, this is not required. Text to use within the notification banner. If `html` is provided, the `text` argument will be ignored.
+- name: html
+  type: string
+  required: true
+  description: If `text` is set, this is not required. HTML to use within the notification banner. If `html` is provided, the `text` argument will be ignored.
```

Comment:
Could personalise this and phrase it in the active voice: If you set 'text,' this...

Reply:
> Do we say 'parameter' instead of 'argument'?

We actually say 'option' 😅  but this might not always be consistent throughout the rest of the tech docs.